Dockerfile.daemon.openshift

```diff
@@ -1,4 +1,8 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12 AS builder
+RUN yum install -y libbpf libbpf-devel libxdp-1.2.1-1.el8.x86_64
```

Comment:
we should be able to install libbpf and libbpf-devel but not lixdp right ?

Reply:
libxdp-devel requires libxdp and it'll pull it in as a dependency (the .o files are also required as we have a dep on libxdp.h)